src/components/pages/Quiz/StateQuestion.js

```diff
@@ -40,12 +40,6 @@ const StateQuestion = ({ quiz, setLocation }) => {
         </Typography>
       </Box>
 
-      <Box mb={1}>
-        <Typography variant="body1">
-          Select the state you live in (the state where the harm occurred).
-        </Typography>
-      </Box>
-
```

Comment:
@sarahgreen just a heads up that I'm removing this entirely because the question title says "Question 1 of 8: Where do you live?" so no need to ask the user this question twice.